.github/workflows/simorgh-local-server-tests.yml

```
@@ -52,14 +52,12 @@ jobs:
         working-directory: ./ws-nextjs-app
         run: PORT=7081 nohup node build/standalone/ws-nextjs-app/server.js &
 
-      - name: Run AMP Validator
-        run: yarn amp:validate
+      # TEMP: fail faster on CI
+      # - name: Run AMP Validator
+      #   run: yarn amp:validate
 
-      - name: Run Lighthouse
-        run: yarn lighthouse
+      # - name: Run Lighthouse
+      #   run: yarn lighthouse
```

Don't forget to put this back!

```suggestion
       - name: Run AMP Validator
         run: yarn amp:validate
       - name: Run Lighthouse
         run: yarn lighthouse
```